### ISSUE TITLE
Start server and client at boot (Raspberry Pi image)

### ISSUE THREAD

**Issue by andrewerrington** [pagermon/pagermon#518]
I have successfully installed the Pi image on a Raspberry Pi Zero 2 W and it is working with my TV dongle to receive POCSAG messages.

I followed the instructions here:
https://github.com/pagermon/pagermon/wiki/PagermonPi-Image-For-Raspberry-Pi

There are no instructions to make the client and server start at boot time.

The main GitHub page for this project (https://github.com/pagermon/pagermon) suggests

sudo pm2 startup
pm2 save

for the server, but nothing for the client.

**To Reproduce**
Install and configure the Pagermon Pi image. Get everything working. Reboot the Pi (e.g. with sudo reboot)

**Expected behavior**
Pi should boot up. Server and client should start. Web page should be visible from any browser on network. New messages should appear in the list as they are captured.

What actually happens is that

pm2 status

shows no processes running, an Pagermon is inaccessible (because it's not running).

I'm not sure it's a bug report. Maybe it's a bug in the documentation, because it would seem that pm2 is capable of starting the processes at boot, but it doesn't seem to be happening.

I'm happy to test any instructions.

**Comment by andrewerrington** [comment]
Ok. It turns out that

pm2 startup

prints a message with some code that needs to be executed to make the startup function actually work. I copied and pasted what it gave me, then rebooted.

Server and client were running after reboot. Success!

I'll leave this issue here for a short time, in case I'm doing it wrong, or if there is more I should be aware of, but it would be helpful to update the documentation to indicate this vital step (i.e. it's insufficient just to run pm2 startup- you have to do what it tells you).

**Comment by andrewerrington** [comment]
Ok. No further comments from anyone. This topic will show up if anyone searches. I'm happy, having tested it precisely once.

tl;dr read and follow the output of _pm2 startup_
